tools/msgpack4s/src/main/scala/org.velvia/msgpack/PlayJsonCodecs.scala

```diff
@@ -28,16 +26,33 @@ object PlayJsonCodecs {
   }
 
   implicit object JsNumberCodec extends Codec[JsNumber] {
-    def pack(out: DataOutputStream, item: JsNumber): Unit = { BigDecimalCodec.pack(out, item.value.underlying) }
-    val unpackFuncMap = BigDecimalCodec.unpackFuncMap.mapValues(_.andThen(b ⇒ JsNumber(BigDecimal(b))))
+    def pack(out: DataOutputStream, item: JsNumber): Unit = {
+      val value: BigDecimal = item.value
+
```

Comment:
@dmitriyisaev , please check bellow if statements for ability to make it more "scala" style?